src/Weikio.ApiFramework.Core/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -186,6 +187,10 @@ public static class ServiceCollectionExtensions
                 mvcOptions.Filters.Add(filter);
             });
 
+            services.AddDistributedMemoryCache();
```

Comment:
The DistributedMemoryCache should be added only if it isn't already configured in the app. Make sure (by checking the source code of AddDistributedMemoryCache) that this is skipped if configuration has already happened before this line.

Reply:
`AddDistributedMemoryCache()` uses `TryAdd` internally but it's not clear to the reader without checking the source code. 

Made this more explicit by wrapping the statement in if-clause which checks the existence of service type `IDistributedCache`:

```
if (!services.Any(s => s.ServiceType == typeof(IDistributedCache)))
{ 
    services.AddDistributedMemoryCache();
}
```